app/yaba.yaml

```
@@ -35,7 +35,7 @@ paths:
                 CSV:
                   summary: A CSV file for experiment metadata Upload.Below is the schema of csv file with example row.
                   text/csv: |-
-                    name,start_date,end_date,description,design
+                    name,start_date,end_date,description
                     'MAC Season 8: Border','2019-01-01','2019-03-31','ProMix BRK20 14-14-14 Osmocote pots; pre-filled by Hummert Sorghum seed Conviron Growth House'....,'Grain Sorghum also planted to fill out empty plots'
```

```suggestion
                    'MAC Season 8: Border','2019-01-01','2019-03-31','ProMix BRK20 14-14-14 Osmocote pots; pre-filled by Hummert Sorghum seed Conviron Growth House'
```
I guess this must be removed too.